libyul/backends/wasm/BinaryTransform.cpp

```diff
@@ -381,30 +376,21 @@ bytes BinaryTransform::operator()(BuiltinCall const& _call)
 		return toBytes(Opcode::I64Const) + lebEncodeSigned(static_cast<int64_t>(m_subModulePosAndSize.at(name).second));
 	}
 
+	yulAssert(builtins.count(_call.functionName), "Builtin " + _call.functionName + " not found");
+	// NOTE: the dialect ensures we have the right amount of arguments
 	bytes args = visit(_call.arguments);
+	bytes ret = move(args) + toBytes(builtins.at(_call.functionName));
```

Comment:
Should this be ?
```suggestion
	bytes ret = visit(_call.arguments) + toBytes(builtins.at(_call.functionName));
```

Reply:
I don't think it makes a difference, but having a call to `visit` stand out is probably a good thing - also concerning the order of evaluation.

Reply:
> also concerning the order of evaluation.

That's why I left it this way, but a lot of other places in the code actually have it in a single statement.